### ISSUE TITLE
Feature Request: "Create new group on password save"

### ISSUE THREAD

**Issue by davidfaber** [buttercup/buttercup-browser-extension#273]
It would be awesome to have the possibility to create new Groups, when saving a password. Now I always have to go the desktop app, create the group, wait until it syncs (takes forever) and then save the password. Or I save the password to a group where it doesn't belong, go to the desktop app, wait until it syncs (again... takes forever) and move over the entry to a new group.

Thanks so much for your awesome tools!

**Comment by perry-mitchell** [comment]
@davidfaber Good point, we need a way to save new groups. However it's already possible to do all of this in the extension, without going to the desktop app. Simply locate your vault in the menu and click it - it'll open the full vault editing interface in the browser so you can create a group or move entries, etc..
